lms/views/dashboard/api/assignment.py

```diff
@@ -179,7 +180,7 @@ def course_assignments_metrics(self) -> APIAssignments:
                 metrics = AnnotationMetrics(
                     annotations=h_stats["annotations"] + h_stats["page_notes"],
                     replies=h_stats["replies"],
-                    last_activity=h_stats["last_activity"],
+                    last_activity=datetime.fromisoformat(h_stats["last_activity"]),
```

Comment:
This are not dates from our database but from H's API.

We could also fix this by returning the tz info from H but this change in LMS will also work if we eventually do that.

Reply:
I guess we'll eventually want to "normalize" ISO dates in H API as well, regardless of the issue being solved at present, but for now this should do.

Reply:
Yup, exactly, with this change we won't need to do any further change in LMS so there's that.